fabfile.py

```diff
@@ -1544,6 +1545,19 @@ def install_java():
 
 
 @task
+def install_yarn():
+    """Install yarn"""
+    if not env.mac:
+        if not exists('/etc/apt/sources.list.d/yarn.list'):
```

Comment:
you already install an updated node version via nodeenv if you don't have a recent node version on the system, see update_node task in fabfile.py